local-devel/base.cfg

```diff
@@ -0,0 +1 @@
+./plone/base.cfg
```

Comment:
I'm not sure I've got this one. Do I have to manually `git clone` something here?

Reply:
We [list the buildout configuration files from the base Plone image](https://github.com/eea/eea.docker.plone/blob/build-local-devel/local-devel/Makefile#L49) then we [extract those files into that `./local-devel/plone/` directory](https://github.com/eea/eea.docker.plone/blob/build-local-devel/local-devel/Makefile#L162https://github.com/eea/eea.docker.plone/blob/build-local-devel/local-devel/Makefile#L162) and finally we [symlink them to files like these as appropriate](https://github.com/eea/eea.docker.plone/blob/build-local-devel/local-devel/Makefile#L152).  Did you try running `$ make` and the target of that link still wasn't in place?

Reply:
He he, "it doesn't work on my machine" :smile: 
```
sudo: apt-get: command not found
```

Reply:
OS X, I take it?

Reply:
Thanks for being my first real tester!  ;-)  I've added some more informative error message when not run on Debian/Ubuntu.  So please do test again if you have time.

Reply:
> OS X, I take it?

ArchLinux

Reply:
Moved to [another PR](https://github.com/eea/eea.docker.plone/pull/5)